src/utils/transitions.js

```diff
@@ -1,5 +1,5 @@
 export const mobileMinWidth = 0;
-export const desktopMinWidth = 1200;
```

Comment:
desktopMinWidth is still `1200`, this PR introduces a new breakpoint for tablets, you should add a new const called `tabletMinWidth = 840` and use that within the responsive transition in the modal component.